### PR TITLE
squashfs-tools-ng: 1.0.4 -> 1.1.0

### PR DESCRIPTION
**Pull request by alyssais**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/filesystems/squashfs-tools-ng/default.nix [hunk 1]
```diff
@@ -1,17 +1,17 @@
-{ stdenv, lib, fetchurl, doxygen, graphviz, perl, pkg-config
+{ stdenv, lib, fetchurl, doxygen, graphviz, perl, pkgconfig
```
Comment (SuperSandro2000) [inline pkgs/tools/filesystems/squashfs-tools-ng/default.nix:1]: Please revert that change as it fails ofborg no aliases check.

File: pkgs/tools/filesystems/squashfs-tools-ng/default.nix [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
+  version = "1.1.0";
 
   src = fetchurl {
     url = "https://infraroot.at/pub/squashfs/squashfs-tools-ng-${version}.tar.xz";
-    sha256 = "04zvpws1nk3r2kr1k5in9di9fhn4zzciyndgnxnijmhiqpyrx072";
+    sha256 = "1swsw5j8rrjxdxsfyd446f6g8f0k3mwg15baivi953i69c9981qi";
   };
 
-  nativeBuildInputs = [ doxygen graphviz pkg-config perl ];
+  nativeBuildInputs = [ doxygen graphviz pkgconfig perl ];
```
Comment (SuperSandro2000) [inline pkgs/tools/filesystems/squashfs-tools-ng/default.nix:14]: same here.

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]
other than that the diff LGTM

**Comment by alyssais** [comment]
Oh, thanks for catching the pkgconfig thing -- I wrote the patch in an
old version of Nixpkgs and then copied it to a recent one.  Oops. :)

**Comment by alyssais** [comment]
@SuperSandro2000 would you mind checking whether this is still broken on Darwin?

**Comment by SuperSandro2000** [comment]
> @SuperSandro2000 would you mind checking whether this is still broken on Darwin?

yeah
```
Undefined symbols for architecture x86_64:
  "_utimensat", referenced from:
      _set_attribs in rdsquashfs-restore_fstree.o
ld: symbol(s) not found for architecture x86_64
```

**Pull request merged**